tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/16-layout/60-split/Split_Layout.test.js

```diff
@@ -0,0 +1,109 @@
+  let verticalSplitters = getVerticalSplitters("#page\\:mainForm\\:Example1");
+  let doubleSplitters = hasDoubleSplitter("#page\\:mainForm\\:Example1");
+
+  const test = new JasmineTestTool(done);
+  test.do(() => expect(horizontalSplitters === expectedSplitters).toBe(true));
+  test.do(() => expect(verticalSplitters === 0).toBe(true));
+  test.do(() => expect(doubleSplitters === false).toBe(true));
+  test.start();
+});
+
+it("SplitLayout Example 2", function (done) {
+  let expectedSplitters = getElements("#page\\:mainForm\\:Example2") - 1;
+  let horizontalSplitters = getHorizontalSplitters("#page\\:mainForm\\:Example2");
+  let verticalSplitters = getVerticalSplitters("#page\\:mainForm\\:Example2");
+  let doubleSplitters = hasDoubleSplitter("#page\\:mainForm\\:Example2");
+
+  const test = new JasmineTestTool(done);
+  test.do(() => expect(horizontalSplitters === expectedSplitters).toBe(true));
+  test.do(() => expect(verticalSplitters === 0).toBe(true));
+  test.do(() => expect(doubleSplitters === false).toBe(true));
+  test.start();
+});
+
+it("SplitLayout Example 3a", function (done) {
+  let expectedSplitters = getElements("#page\\:mainForm\\:Example3a") - 1;
+  let horizontalSplitters = getHorizontalSplitters("#page\\:mainForm\\:Example3a");
+  let verticalSplitters = getVerticalSplitters("#page\\:mainForm\\:Example3a");
+  let doubleSplitters = hasDoubleSplitter("#page\\:mainForm\\:Example3a");
+
+  const test = new JasmineTestTool(done);
+  test.do(() => expect(horizontalSplitters === expectedSplitters).toBe(true));
+  test.do(() => expect(verticalSplitters === 0).toBe(true));
+  test.do(() => expect(doubleSplitters === false).toBe(true));
+  test.start();
+});
+
+it("SplitLayout Example 3b", function (done) {
+  let expectedSplitters = getElements("#page\\:mainForm\\:Example3b") - 1;
+  let horizontalSplitters = getHorizontalSplitters("#page\\:mainForm\\:Example3b");
+  let verticalSplitters = getVerticalSplitters("#page\\:mainForm\\:Example3b");
+  let doubleSplitters = hasDoubleSplitter("#page\\:mainForm\\:Example3b");
+
+  const test = new JasmineTestTool(done);
+  test.do(() => expect(horizontalSplitters === 0).toBe(true));
+  test.do(() => expect(verticalSplitters === expectedSplitters).toBe(true));
+  test.do(() => expect(doubleSplitters === false).toBe(true));
+  test.start();
+});
+
+function getHorizontalSplitters(element) {
+  return querySelectorFn(element)().querySelectorAll(":scope > .tobago-splitLayout-horizontal, :scope > .tobago-splitLayout-horizontal").length;
+}
+
+function getVerticalSplitters(element) {
+  return querySelectorFn(element)().querySelectorAll(":scope > .tobago-splitLayout-vertical, :scope > .tobago-splitLayout-vertical").length;
+}
+
+function hasDoubleSplitter(element) {
+  let counter = 0;
+  let elements = querySelectorFn(element)().children;
+
+  for (let item of elements) {
+    if(item.className === "tobago-splitLayout-horizontal" || item.className === "tobago-splitLayout-vertical") {
```

Comment:
Use item.classList to check if a CSS class is set.
className return the whole string of the 'class' attribute.